### ISSUE TITLE
Discord

### ISSUE THREAD

**Issue by darkhershel** [Treeofsavior/PortugueseTranslation#107]
Vamos nos juntar no DISCORD para melhorar isso... 
Lá poderemos dividir melhor quem vai ficar com qual parte em tempo real... 
https://discord.gg/0aMoaV0VhjXiGTiS

**Comment by darkhershel** [comment]
e discutir outras coisas... 
como ficará nome de locais e monstros, por exemplo

**Comment by faicciruz** [comment]
Link expirado.

**Comment by darkhershel** [comment]
sim... quando eu coloquei ninguem fez questão de entrar.... EU nem atualizei o link

**Comment by darkhershel** [comment]
https://discord.gg/0aMoaV0VhjVbo4yj

**Comment by faicciruz** [comment]
Perdi o invite, desculpa. Estou fazendo parte do canal do Discord inglês. Pode me mandar o convite de novo?

**Comment by darkhershel** [comment]
https://discord.gg/0aMoaV0VhjVUljlz
